Make sure tutorial will work for user (#173)
libp2p/docs: content/tutorials/getting-started/javascript.md

```

Success! Our two peers are now communicating over a multiplexed, secure channel.  Sure, they can only say "ping", but it's a start!

> ## Working version

Reviewer: ```suggestion
## Working versions

```

> This tutorial was created and tested using these packages versions:

Reviewer: ```suggestion
This tutorial uses the following js-libp2p package versions:
```

```
    "@chainsafe/libp2p-noise": "^8.0.1",
    "@libp2p/mplex": "^5.1.0",
    "@libp2p/tcp": "^3.0.4",
    "@libp2p/websockets": "^3.0.2",
    "libp2p": "^0.38.0",
    "multiaddr": "^10.0.1"
```

> If you are using any different version of those above, make sure to change your code and adapt it to work for you.  

Reviewer: ```suggestion

> Modify your js-libp2p packages if you start to experience issues to those mentioned above.
```
